### ISSUE TITLE
Incorrect optional redirect_uri value

### ISSUE THREAD

**Issue by AlekzandrV** [Nebo15/mithril.api#37]
Optional attribute must be Nil

Request:
```
curl -k  -L 'http://35.187.76.226/admin/clients/28aa6d09-6727-4dbb-bc59-7bd906cfd113' | jq
```

Response:
```
{
   "meta":{
      "url":"http://35.187.76.226/admin/clients/28aa6d09-6727-4dbb-bc59-7bd906cfd113",
      "type":"object",
      "request_id":"2nstu94v7djt8a07l05m35qb2aarvces",
      "code":200
   },
   "data":{
      "settings":{

      },
      "secret":"U2RIcDZSY09td0FVaDVWNXZFSTVrUT09",
      "redirect_uri":"fill_me",
      "priv_settings":{

      },
      "name":"testClient3",
      "id":"28aa6d09-6727-4dbb-bc59-7bd906cfd113",
      "type":"client"
   }
}
```

**Comment by gmile** [comment]
We agreed that `redirect_uri` absolutely cannot be avoided during creation.

I will revert the change that made `redirect_uri` optional, which in turn will make this ticket obsolete.

**Comment by gmile** [comment]
Here, `settings` and `priv_settings` will default to `{}` (blank json), and not nil. `redirect_uri` will no longer default to `fill_me`, as it is always required and will be set to whatever value is passed from requester.

**Comment by gmile** [comment]
Resolved by 6bc238647728779b1885e0639ed5a88d0578593f
